projects/wave-core/src/lib/operators/dialogs/data-repository/raster/source-dataset.component.ts

```diff
@@ -26,15 +26,15 @@ import {Colormap} from '../../../../colors/colormaps/colormap.model';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 
-export class SourceDatasetComponent implements OnInit {
+export class SourceDatasetComponent implements OnInit , OnChanges {
```

Comment:
Same here.

Reply:
formated in 6da1d4c74b7a75a702074ebf5ef83c4bda092d4f